src/printer.js

```diff
@@ -3,6 +3,8 @@
 const docBuilders = require("prettier").doc.builders;
 const docUtils = require("prettier").doc.utils;
 const sharedUtil = require("prettier").util;
+//@TODO: super hacky, remove once comment functions are exported for plugin use
+const comments = require("../node_modules/prettier/src/main/comments");
```

Comment:
https://github.com/prettier/prettier/issues/4177

Reply:
Isn't this fixed in prettier core now?

Reply:
Apart from this, this looks all reaaally good! 💯